Gems/Multiplayer/Code/Include/Multiplayer/MultiplayerTypes.h

```diff
@@ -32,6 +32,12 @@ namespace Multiplayer
     AZ_TYPE_SAFE_INTEGRAL(NetEntityId, uint64_t);
     static constexpr NetEntityId InvalidNetEntityId = static_cast<NetEntityId>(-1);
 
+    using NetEntityIdSet = AZStd::unordered_set<NetEntityId>;
+
+    //! The maximum number of netEntityIds we can stuff into a single reset packet
+    static const uint32_t MaxAggregateEntityResets = 2048;
```

Comment:
minor: IMHO we should add hiding constraints in the middle of headers like this. Consider how dev will find and modify, unless this is a true MAX constraint.